vcpkg/overlay/json-c/portfile.cmake

```diff
@@ -0,0 +1,24 @@
+vcpkg_from_github(

```

Comment:
This is needed due to the following failure: https://github.com/opengisch/QField/actions/runs/3065670252/jobs/4950004479